docker-compose.yml

```diff
@@ -6,9 +6,14 @@ services:
       - "5432:5432"
     environment:
       POSTGRES_HOST_AUTH_METHOD: "trust"
-  elasticsearch:
-    # image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.13
+  opensearch:
+    container_name: opensearch
+    restart: always
+    image: opensearchproject/opensearch:2.15.0
```

Comment:
Doesn't matter too much but if you're going for PROD parity we're on 2.13.  AWS said they supported 2.15 but they didn't have it available at time of upgrade.